pyaedt/application/analysis_hf.py

```diff
@@ -360,3 +360,64 @@ def export_touchstone(
             impedance=impedance,
             comments=gamma_impedance_comments,
         )
+
+
+def phase_expression(m, n, theta_name="theta_scan", phi_name="phi_scan"):
+    """Return an expression for the source phase angle in a rectangular antenna array.
+
+    Parameters
+    ----------
+    m : int, required
+        Index of the rectangular antenna array element in the x-direction.
+    n : int, required
+        Index of the rectangular antenna array element in the y-direction.
+    theta_name : str, optional
+        Post-processing variable name in HFSS that is used for the
+        theta component of the phase angle expression. Default is ``"theta_scan"``.
+    phi_name : str, optional
+        Post-processing variable name in HFSS to be used to generate
+        the phi component of the phase angle expression. Default is ``"phi_scan"``
```

Comment:
```suggestion
    m : int, required
        Index of the rectangular antenna array element in the x direction.
    n : int, required
        Index of the rectangular antenna array element in the y direction.
    theta_name : str, optional
        Postprocessing variable name in HFSS to use for the
        theta component of the phase angle expression. The default is ``"theta_scan"``.
    phi_name : str, optional
        Postprocessing variable name in HFSS to use to generate
        the phi component of the phase angle expression. The default is ``"phi_scan"``
```
We usually only put "optional," with the assumption that a parameter is required if "optional" isn't present. I'm not sure we use "require" anywhere elese.